backend/middleware/audit/audit_test.go

```diff
@@ -108,3 +109,78 @@ func TestInterceptorShortCircuitDisabled(t *testing.T) {
 	assert.EqualValues(t, 0, a.writeCount)
 	assert.EqualValues(t, 0, a.updateCount)
 }
+
+func TestInterceptorResponseWithRedaction(t *testing.T) {
```

Comment:
Should we have a test for the request copy as well?

Reply:
Yeah actually I can change this test to do both, let me do that.